src/main/java/dev/andrylat/vomelianchuk/card_number_operations/CardNumber.java

```diff
@@ -14,7 +18,11 @@ public static void main(String[] args) {
                 CardNumberValidator cardNumberValidator = new CardNumberValidator();
                 PaymentSystem paymentSystem = cardNumberValidator.validate(cardNumberAsString);
                 userConsoleIO.printPaymentSystem(paymentSystem);
-            } catch (CardValidationException | WrongPaymentSystemException e) {
+
```

Comment:
So, when your application will have 5-10 modules, you will have here Christmas tree of catches, right?